src/game.h

```diff
@@ -493,6 +493,8 @@ class game
         /** Custom-filtered menu for inventory items and those that are nearby (within @ref radius). */
         item_location inv_map_splice( item_filter filter, const std::string &title, int radius = 0,
                                       const std::string &none_message = "" );
+        /** Book reading menu. */
+        item_location inv_for_books( const std::string &title );
```

Comment:
This mixture reflects the fact that the menu is shown for either one item (gunmod) or many (books).
See `inv_for_activatables()`, `inv_for_tools_powered_by()`, e.t.c.